web/webkit/src/main/scala/net/liftweb/http/LiftRules.scala

```diff
@@ -1321,6 +1321,18 @@ class LiftRules() extends Factory with FormVendor with LazyLoggable {
   val snippets = RulesSeq[SnippetPF]
 
   /**
+   * Handles the parsing of template content into NodeSeqs.
+   */
+  @volatile var contentParsers:Seq[ContentParser] = Seq(
+    new ContentParser {
```

Comment:
Can't we just use `ContentParser.full` here? (Or `.apply` if we switch to that.)